catalog/ui/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerCollection.controller.js

```diff
@@ -60,6 +60,20 @@ define(['underscore',
                 }
 
                 var provider = new type(initObj);
+
+                /*
+                  Optionally add this provider as a TrustedServer. This sets withCredentials = true
+                  on the XmlHttpRequests for CORS.
+                 */
+                if (model.get('withCredentials')) {
```

Comment:
Can we not just always add all layers to cesiums trusted servers? How is openlayers working?

Reply:
I think if it doesn't expect credentials it won't work, so we need to be explicit.

Reply:
@djblue, @mojogitoverhere tried that originally but as @andrewkfiedler points out, it then broke on other servers when withCredentials=true was added to the XmlHttpRequest object.  That is why we strayed back to explicitly defining them